css/obBase.css

```diff
@@ -76,10 +76,24 @@ textarea {
   font-weight: 500;
 }
 
+input[type="file"] {
+    padding-left:10px;
+    padding-top:15px;
+    color: #fff;
+}
```

Comment:
Remove this, instead add type="file" to the section that starts on line 1219, with the input styles that are specific to fieldItem elements.

The fieldItem input style already has the styles needed, it just needs the file type added to it.